src/utils/configurationVault.js

```diff
@@ -1,12 +1,31 @@
 // @flow
+import fs from 'fs'
 import Conf from 'conf'
 
 import {
   CONFIGURATION_PROMPT_NAMES,
   EMOJI_COMMIT_FORMATS
 } from '../commands/config/prompts'
 
-export const config = new Conf({ projectName: 'gitmoji' })
+const localConfigFile = '.gitmojirc';
+let hasLocalConfig = false;
+try {
+  fs.accessSync(localConfigFile, fs.constants.F_OK)
+  hasLocalConfig = true;
+} catch (err) {
+  //local config file does not exist
+}
+const confOptions = hasLocalConfig 
+  ? {
+    cwd: process.cwd(),
+    configName: localConfigFile,
+    fileExtension: undefined
+  }
+  : {
+    projectName: 'gitmoji'
+  };
+
```

Comment:
I would set this into a function like:

```suggestion
const getConfiguration = () => {
  try {
    // We should validate in some way the configuration options obtained from the file/package.json
    if (fs.existsSync(GITMOJI_CONFIGURATION_OPTIONS)) {
      return {
        cwd: process.cwd(),
        configName: localConfigFile,
        fileExtension: undefined
      }
    }

    // Get configuration options from package.json if exist apply them

    return { projectName: 'gitmoji' }
  } catch (error) {
    
  }
}
```